music/demos/index.html

```diff
@@ -41,6 +41,7 @@ <h3>
         <li><a href="music_vae.html">Generate new NoteSequences using MusicVAE</a></li>
         <li><a href="multitrack.html">Multitrack MusicVAE</a></li>
         <li><a href="transcription.html">Piano transcription with Onsets and Frames</a></li>
+        <li><a href="piano_genie.html">Piano Genie</a></li>
```

Comment:
Can you add another like 3 or 4 words describing what this is (like we did for the other models)